src/main/java/com/raoulvdberge/refinedstorage/apiimpl/autocrafting/task/CraftingTask.java

```diff
@@ -535,6 +547,8 @@ public NBTTagCompound writeToNBT(NBTTagCompound tag) {
                             element = new CraftingMonitorElementError(element, "gui.refinedstorage:crafting_monitor.machine_none");
                         } else if (!step.hasStartedProcessing() && !step.canStartProcessing()) {
                             element = new CraftingMonitorElementError(element, "gui.refinedstorage:crafting_monitor.machine_in_use");
+                        } else if (!step.hasStartedProcessing() && step.isBlocked()) {
+                            element = new CraftingMonitorElementError(element, "gui.refinedstorage:crafting_monitor.blocked");
```

Comment:
I can see why you would need `step.isBlocked()` but is `Machine in use` not kinda the same? imo not really needed. paging @raoulvdberge for this one